docker-compose/nested-spire/scripts/set-env.sh

```diff
@@ -82,7 +82,8 @@ docker-compose -f "${PARENT_DIR}"/docker-compose.yaml exec -T root-server \
     -spiffeID "spiffe://example.org/nestedA" \
     -selector "docker:label:org.example.name:nestedA-server" \
     -downstream \
-    -ttl 3600
+    -x509SVIDTTL 3600 \
```

Comment:
I think we should drop these since they are the default for X509-SVID and JWT-SVID is not used AFAIK.